config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   kubernetes-sigs/node-feature-discovery:
   - name: pull-node-feature-discovery-verify
-    always_run: true
+    skip_if_only_changed: "^docs/|^demo/|^deployment/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
```

Comment:
We definitely want to verify `deployment/` (helm lint). We could keep `demo/` as well

Reply:
fixed, thanks.